lib/helpers/LanguageUiHelper.ts

```diff
@@ -0,0 +1,88 @@
+import {Page, Locator, expect} from "@playwright/test"
+import {UiBaseLocators} from "./UiBaseLocators";
+
+export class LanguageUiHelper extends UiBaseLocators{
+  private readonly languageRoot: Locator;
+  private readonly createMenu: Locator;
+  private readonly addLanguageBtn: Locator;
+  private readonly languageDropdown: Locator;
+  private readonly defaultLanguageToggle: Locator;
+  private readonly manatoryLanguageToggle: Locator;
+  private readonly addFallbackLanguageBtn: Locator;
+  private readonly confirmToRemoveBtn: Locator;
+  private readonly deleteLanguageBtn: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.languageRoot = page.locator('umb-menu').getByLabel('Language');
+    this.createMenu = page.locator('umb-entity-action').getByLabel('Create', {exact: true});
+    this.addLanguageBtn = page.getByLabel('Add language', {exact: true});
+    this.languageDropdown = page.locator('umb-input-culture-select #expand-symbol-wrapper');
+    this.defaultLanguageToggle = page.locator('uui-toggle').filter({ hasText: /Default language/ }).locator('#slider');
+    this.manatoryLanguageToggle = page.locator('uui-toggle').filter({ hasText: /Mandatory language/ }).locator('#slider');
+    this.addFallbackLanguageBtn = page.locator('#add-button');
+    this.confirmToRemoveBtn = page.locator("#confirm").getByLabel("Remove");
+    this.deleteLanguageBtn = page.getByRole('button', { name: 'Delete' });
+  }
+
+  async clickLanguageRoot() {
+    await this.languageRoot.click();
+  }
+
+  async clickCreateMenu() {
+    await this.createMenu.click();
+  }
+
+  async clickAddLanguageButton() {
+    await this.addLanguageBtn.click();
+  }
+
+  async removeFallbackLanguageByName(name: string) {
+    await this.page.locator('uui-ref-list').getByLabel('Remove ' + name, {exact: true}).click();
+    await this.confirmToRemoveBtn.click();
+  }
+
+  async chooseLanguageByName(name: string) {
+    await this.languageDropdown.click();
+    await this.page.locator('umb-input-culture-select').getByText(name, {exact: true}).click();
+  }
+
+  async clickLanguageByName(name: string) {
+    await this.page.locator('umb-language-root-table-name-column-layout').getByText(name, {exact: true}).click();
+  }
+
+  async isLanguageNameVisible(name: string) {
+    return await expect(this.page.locator('umb-language-root-table-name-column-layout').getByText(name, {exact: true})).toBeVisible();
+  }
+
+  async isLanguageNameNotVisible(name: string) {
+    return await expect(this.page.locator('umb-language-root-table-name-column-layout').getByText(name, {exact: true})).not.toBeVisible();
+  }
```

Comment:
Having two methods for checking visibility is not necessary. We can have a method that takes a boolean instead
```suggestion
  async isLanguageNameVisible(name: string, isVisible = true) {
    return await expect(this.page.locator('umb-language-root-table-name-column-layout').getByText(name, {exact: true})).toBeVisible({visible: isVisible});
  }
```